### PR TITLE
 Open Source License Acknowledgements and Third-Party Copyrights

### PR DESCRIPTION
**Pull request by petrsvob**
Changes for the compliance with the law and for the consistency with our Commercial License.

Modifications according to compliance with the law + example of added copyrights for first 3 items + correction of the Grunt license URL

TODO:
- [ ] copyrights for the rest of items have to be added (you can typically find them in the License text or in the Copyright or the Notice file)
- [x] rename the file to "open-source-license-acknowledgements-and-third-party-copyrights.md"

| Q             | A
| ------------- | ---
|Description, reason for the PR| Changes for the compliance with the law
|New feature| No
|BC breaks| No
|Fixes issues| ...
|Standards and tests pass| Don't know
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

File: license-overview-of-included-3rd-party-libraries.md [hunk 1]
```diff
@@ -243,4 +247,4 @@ As a source of information about licenses of images and libraries downloaded and
 
 Sources of information about licenses of libraries and components that are not downloaded and installed dynamically are the source files of libraries itself or the GitHub repositories of these libraries.
 
-The transitive dependencies of the dependencies and images of 3rd parties are not included.
\ No newline at end of file
+The transitive dependencies of the dependencies and images of 3rd parties are not included.
```
Comment (petrsvob) [inline license-overview-of-included-3rd-party-libraries.md]: I haven't changed anything here.

**Comment by boris-brtan** [comment]
squashed and rebased

**Comment by boris-brtan** [comment]
renamed file and its prelinks

**Comment by boris-brtan** [comment]
shouldn't this be somehow automated? looks like these years in copyright are dynamic. @TomasVotruba have you met with something like this?

**Comment by TomasVotruba** [comment]
@boris-brtan Good question! It mostly about personal preference, just like tabs and spaces.

- Symfony: https://github.com/symfony/symfony/commit/07a7f560cdfb7d0b6984b03231b1280f8420d41e
- Nette: 24620a5f8e66dcb03772af3b5fa2847826c0f10c

I prefer the later, because the origin year is clear and people usually know what year it is. Also, there are better thing to do to use one's life time

But in the end it depends on personal preference

File: README.md [hunk 2]
```diff
@@ -107,4 +107,4 @@ Learn the principles on which we distribute our product on our website at [Licen
 The rest of modules of Shopsys Framework including [HTTP smoke testing](https://github.com/shopsys/http-smoke-testing) are distributed under standard MIT license. 
 
 Shopsys Framework also uses some third-party components and images which are licensed under their own respective licenses.
-The list of these licenses is summarized in [License overview of included 3rd party libraries](./license-overview-of-included-3rd-party-libraries.md).
+The list of these licenses is summarized in  [Open Source License Acknowledgements and Third Party Copyrights](./open-source-license-acknowledgements-and-third-party-copyrights.md).
```
Comment (LukasHeinz) [inline README.md]:
```suggestion
The list of these licenses is summarized in [Open Source License Acknowledgements and Third Party Copyrights](./open-source-license-acknowledgements-and-third-party-copyrights.md).
```

Comment (LukasHeinz) [reply README.md]: Too much spaces

Comment (boris-brtan) [reply README.md]: THX

**Pull request merged**